client/src/components/ratingsReviews/ratings.jsx

```diff
@@ -56,8 +63,9 @@ class Reviews extends React.Component {
     renderReviews() {
         let reviews = this.state.allReviews;
```

Comment:
Try to use Const instead of let for variables that will not be changed directly.